src/PortalWithState.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { findDOMNode } from 'react-dom';
 import PropTypes from 'prop-types';
-import Portal from './Portal';
+import Portal from './PortalUtil';
```

Comment:
Can we rename `PortalUtil` to `PortalCompat`. Or anyone else having a better idea?

Reply:
👍 `PortalCompat` seems like a much better name, let's go with that unless someone else comes up with a better one